content/tracing/faq/traces-sampling-and-storage.md

```diff
@@ -0,0 +1,112 @@
+
+**Priority Sampling is an additional sampling option in which you indicate whether a trace should be kept. Priority Sampling runs prior to Signature Sampling, and a decision is made at the beginning of a trace**. As the decision is propagated in the trace context, it has two main properties:
+
+* It ensures completeness of traces distributed across hosts and services
+* Important traces can be kept
+
+#### How it works
+
+Each trace has a `sampling_priority`, which is assigned at its inception and propagated along the entire trace:
+
+| Value      |  Type | Action|
+| :--------------------- | :---------------- | :----------|
+|**-1**|User input| The Agent drops the trace. |
+|**0**|Automatic sampling decision| The Agent drops the trace. |
+|**1**|Automatic sampling decision| The Agent drops the trace. |
+|**2**|User input| The Agent keeps the trace, and the backend will only apply sampling if above maximum volume allowed. |
+
+Traces are automatically assigned a priority of **0** or **1**, with a proportion ensuring that the Agent won't have to sample more than it is allowed.  Override it by assigning a value of **2** if it is a trace that should be kept (critical transaction, debug mode, etc.).  
+
+**Note**: Spans dropped by priority sampler can still be sampled by the signature sampler. The backend can re-sample to keep up to 60 traces per minute per host. The distributed traces that are kept will all be complete. 
+
+#### Using Priority Sampling
+
+Priority Sampling is recommended if your traces are distributed across multiple hosts or if you need finer control over the traces sampled. To use it, enable `distributed_sampling` in your client ([Ruby](http://www.rubydoc.info/gems/ddtrace/#Priority_sampling), [Python](http://pypi.datadoghq.com/trace/docs/#priority-sampling), [Go](https://godoc.org/github.com/DataDog/dd-trace-go/tracer#Span.SetSamplingPriority), [Java](/tracing/setup/java/#sampling-distributed-tracing)) as it is disabled by default.
+
+### Client rate sampling
+
+It is possible to disable the instrumentation for a percentage of transactions. Datadog Tracers are performant and can run with a minimal cost with thousands of requests per second, however client rate sampling can be used to further reduce the instrumentation footprint. In this case, no trace will be generated at all, and Datadog estimates aggregate statistics (requests per second, error rate, etc.). An example of this in Python can be found here: http://pypi.datadoghq.com/trace/docs/#pre-sampling.
+
+## Trace storage
+
+Individual traces are stored for up to 6 months but to determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime. In Datadog backend, sampled traces are retained according to time buckets:
```

Comment:
"Individual traces are stored for up to 6 months but to determine how long a particular trace will be stored~, the Agent makes a sampling decision early in the trace's lifetime~. In Datadog backend, sampled traces are retained according to time buckets:"